File: src/main/java/com/lord/small_box/dtos/DepositControlRequestDto.java
package com.lord.small_box.dtos;


import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DepositControlRequestDto {
private Long id;
	
	private String itemCode;
	
	private String itemMeasureUnit;
	
	private String itemDescription;
	
	
	private long depositRequestId;
}

File: front/small-box/src/app/models/depositControlRequestDto.ts
export class DepositControlRequestDto{
   	id!:number

	itemCode!:string;
	
	itemMeasureUnit!:string;
	
	 itemDescription!:string;
	
     itemQuantity!:number;
	
	 depositRequestId!:number;

	 constructor(itemCode:string,itemMeasureUnit:string,itemDescription:string,itemQuantity:number){
		this.itemCode = itemCode;
		this.itemMeasureUnit = itemMeasureUnit;
		this.itemDescription = itemDescription;
		this.itemQuantity = itemQuantity;
		
	 }
}